programs/uxd/src/utils/math/compute_value_for_shares_amount.rs

```diff
@@ -11,10 +11,15 @@ pub fn compute_value_for_shares_amount(
     if shares_amount == 0 {
         return Ok(0);
     }
-    require!(shares_amount <= total_shares_amount, UxdError::MathError);
-    Ok(shares_amount
+    require!(total_shares_amount > 0, UxdError::MathError);
+    require!(total_shares_value > 0, UxdError::MathError);
```

Comment:
Why `total_shares_value` should be bigger than 0? Would mean we can't mint if we are the first one to do so

Reply:
Well, it's theory, in reality, it will always be the case, because we know which credix depository we are integrated with

Reply:
Yes, its not possible to compute the price of a share if either of those are zeros, we leave the responsibility to the protocol to initialize the pool

Reply:
LGTM